content/en/docs/setup/independent/high-availability.md

```diff
@@ -411,35 +411,41 @@ done
 - Follow [these instructions](/docs/setup/independent/setup-ha-etcd-with-kubeadm/)
    to set up the etcd cluster.
 
-### Copy required files to other control plane nodes
-
-The following certificates were created when you created the cluster. Copy them
-to your other control plane nodes:
-
-- `/etc/kubernetes/pki/etcd/ca.crt`
-- `/etc/kubernetes/pki/apiserver-etcd-client.crt`
-- `/etc/kubernetes/pki/apiserver-etcd-client.key`
+#### Copy required files from an etcd node to all control plane nodes
 
 In the following example, replace `USER` and `CONTROL_PLANE_HOSTS` values with values
 for your environment.
 
 ```sh
+# Make a list of required etcd certificate files
+cat << EOF > etcd-pki-files.txt
+/etc/kubernetes/pki/etcd/ca.crt
+/etc/kubernetes/pki/apiserver-etcd-client.crt
+/etc/kubernetes/pki/apiserver-etcd-client.key
+EOF
+
+# create the archive
+tar -czf etcd-pki.tar.gz -T etcd-pki-files.txt
+
+# copy the archive to the control plane nodes
 USER=ubuntu
 CONTROL_PLANE_HOSTS="10.0.0.7 10.0.0.8 10.0.0.9"
 for host in $CONTROL_PLANE_HOSTS; do
-    scp /etc/kubernetes/pki/etcd/ca.crt "${USER}"@$host:
-    scp /etc/kubernetes/pki/apiserver-etcd-client.crt "${USER}"@$host:
-    scp /etc/kubernetes/pki/apiserver-etcd-client.key "${USER}"@$host:
+    scp etcd-pki.tar.gz "${USER}"@$host:
 done
 ```
 
 ### Set up the first control plane node
 
-1.  Create a `kubeadm-config.yaml` template file:
+1.  Extract the etcd certificates
+
+        tar -xzf etcd-pki.tar.gz -C /etc/kubernetes/pki --strip-components=3
+
+1.  Create a `kubeadm-config.yaml`:
 
         apiVersion: kubeadm.k8s.io/v1alpha2
```

Comment:
the document needs a `v1alpha3` bump.
and also `MasterConfiguration` -> [some-other-configuration]

Reply:
i'm about to send a PR that tries to fix this for some other pages.